account_tax_change/tests/common.py

```diff
@@ -8,8 +8,11 @@
 
 class AccountTaxChangeCommon(AccountTestInvoicingCommon):
     @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
+    def setUpClass(cls, chart_template_ref=None):
+        chart_template_ref = (
+            chart_template_ref or "l10n_generic_coa.configurable_chart_template"
```

Comment:
are you sure we need this default?

Seems to be the default in core https://github.com/odoo/odoo/blob/c0dce31bd509de3b966887f739fa565f3c737720/addons/account/tests/common.py#L37

Reply:
@simahawk  i seen it but somehow even tests for this addon are skipped because template not provided. I think it doesn't harm to have it here if it helps to run tests correctly.